Description:
In eglibc <= 2.9, getline was only defined if _GNU_SOURCE was defined. In
eglibc 2.10, getline is always defined (since it became a standard in
POSIX2008). The uni2ascii already has a function named getline(), which now
conflicts with glibc's. Patch will rename getline() to get_line() to fix the
problem.
Author: Kartik Mistry <kartik@debian.org>
Debian bug: http://bugs.debian.org/552870
--- a/ascii2uni.c
+++ b/ascii2uni.c
@@ -463,7 +463,7 @@
      fprintf(stderr,"Failed to allocate buffer for input line.\n");
      exit(2);
    }
-   while ((read = getline(&lbuf, &len, infp)) != -1) {
+   while ((read = get_line(&lbuf, &len, infp)) != -1) {
      AddNewlineP = 1;
      LineNo++;
      last = read - 1;
@@ -848,7 +848,7 @@
 }
 
 int
-getline (lineptr, n, stream)
+get_line (lineptr, n, stream)
      char **lineptr;
      size_t *n;
      FILE *stream;
